docs/exporters.md

```diff
@@ -4,33 +4,33 @@ Below is a list of supported exports with links to their documentation pages.
-| AWS Kinesis Exporter                                              | [awskinesisexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.57.2/exporter/awskinesisexporter/README.md) |
-| AWS Prometheus Remote Write Exporter                              | [awsprometheusremotewriteexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.57.2/exporter/awsprometheusremotewriteexporter/README.md) |
-| AWS X-Ray Tracing Exporter                                        | [awsxrayexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.57.2/exporter/awsxrayexporter/README.md) |
-| Azure Monitor Exporter                                            | [azuremonitorexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.57.2/exporter/azuremonitorexporter/README.md) |
-| Carbon Exporter                                                   | [carbonexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.57.2/exporter/carbonexporter/README.md) |
-| Elasticsearch Exporter                                            | [elasticsearchexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.57.2/exporter/elasticsearchexporter/README.md) |
-| F5 Cloud Exporter                                                 | [f5cloudexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.57.2/exporter/f5cloudexporter/README.md) |
-| File Exporter                                                     | [fileexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.57.2/exporter/fileexporter/README.md) |
+| Alibaba Cloud Log Service Exporter                                | [alibabacloudlogserviceexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.58.0/exporter/alibabacloudlogserviceexporter/README.md) |
+| AWS CloudWatch Logs Exporter                                      | [awscloudwatchlogsexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.58.0/exporter/awscloudwatchlogsexporter/README.md) |
+| AWS CloudWatch EMF Exporter                                       | [awsemfexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.58.0/exporter/awsemfexporter/README.md) |
+| AWS Kinesis Exporter                                              | [awskinesisexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.58.0/exporter/awskinesisexporter/README.md) |
+| AWS Prometheus Remote Write Exporter                              | [awsprometheusremotewriteexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.58.0/exporter/awsprometheusremotewriteexporter/README.md) |
```

Comment:
Remove this please if we removed the exporter.

Reply:
good catch, removed.